astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 3e044eb26395c8569e3e15bf821c6ae5eeea324a
+Subproject commit fedd7c52ba80e2246f83b8d1752baa3a153a33e2
```

Comment:
Could you please remove this change? I think it was accidental as the helpers are up to date in master.